src/pocapi.cpp

```diff
@@ -568,7 +568,15 @@ enum : unsigned int {
 static Status getModelInstance(ovms::Server& server, const InferenceRequest* request, std::shared_ptr<ovms::ModelInstance>& modelInstance,
     std::unique_ptr<ModelInstanceUnloadGuard>& modelInstanceUnloadGuardPtr) {
     OVMS_PROFILE_FUNCTION();
-    auto& modelManager = dynamic_cast<const ServableManagerModule*>(server.getModule(ovms::SERVABLE_MANAGER_MODULE_NAME))->getServableManager();
+    if (!server.isLive()) {
+        return ovms::Status(ovms::StatusCode::INTERNAL_ERROR, "server is not live");
+    }
+    const ovms::Module* servableModule = server.getModule(ovms::SERVABLE_MANAGER_MODULE_NAME);
+    if (!servableModule) {
+        // TODO: Should we wait for servable manager to appear? And for model to get loaded?
+        return ovms::Status(ovms::StatusCode::INTERNAL_ERROR, "missing servable manager");
```

Comment:
I think here we should check for both server.isLive() & is ready. Then it is not internal error -> it can be usage error since client can call Inference() without starting the server so this is invalid usage.

Then if we pass those two calls (ie server is both live & ready) if we cannot get modelmanager then we have INTERNAL_ERROR

Reply:
done